kuma/static/js/task-completion.js

```diff
@@ -27,7 +29,9 @@
         var clickTime = Date.now() + (1000*60)*20; // 20 min from now
         var surveyURL= 'https://www.surveygizmo.com/s3/2980494/do-or-do-not';
         var surveyLink = surveyURL + '?' + '&t='+ clickTime + '&p=' + path;
-        var ask = "Would you answer 4 questions for us? <a id='task-link' target='_blank' href='" + surveyLink + "'>Open the survey in a new tab</a> and fill it out when you're done on the site. Thanks!";
+        var surveyAttributes = 'id=\'task-link\' target=\'_blank\' href=\'' + surveyLink + '\'';
+        var ask = interpolate(gettext('Would you answer 4 questions for us? <a %(url)s>Open the survey in a new tab</a> and fill it out when you are done on the site. Thanks!'), { url : surveyAttributes }, true);
+
```

Comment:
This extracts, interpolates correctly 👍 ``url`` was an unexpected name for "all the anchor tag attributes", but it works.

Reply:
I was thinking of what the translators will see more than what the devs see ;)